static/js/syntaxModesRules-nl.ts

```diff
@@ -0,0 +1,354 @@
+import * as syntaxRules from './syntaxRules';
+var _PRINT='print';
+var _ASK='vraag';
+var _ECHO='echo';
+var _FORWARD='vooruit';
+var _TURN='draai';
+var _IS='is';
+var _SLEEP='slaap';
+var _AT='op';
+var _RANDOM='willekeurig';
+var _IN='in';
+var _IF='als';
+var _ELSE='anders';
+var _REPEAT='herhaal';
+var _TIMES='keer';
+var _FOR='voor';
+var _RANGE='bereik';
+var _INPUT='invoer';
+/* These keywords currently don't have highlighing rules, but are also listed in the .lark files
+var _OR='of';
+var _WHILE='zolang';
+var _ADD_LIST='voeg';
+var _TO_LIST='toe aan';
+var _REMOVE='verwijder';
+var _FROM='uit';
+var _TO='tot';
+var _STEP='stap';
+var _ELIF='alsanders';
+var _AND='en';
+*/
+
+const LEVELS = [
+  {
+    name: 'level1',
+    rules: syntaxRules.pipe(syntaxRules.baseRules(_AT,_RANDOM),
+      syntaxRules.rule_printSpace(_PRINT,'gobble'),
+      syntaxRules.rule_turtle(_TURN,_FORWARD),
+      syntaxRules.recognize('start', {
+        regex: syntaxRules.keywordWithSpace(_ECHO),
+        token: 'keyword',
+        next: 'gobble',
+      }),
+      syntaxRules.recognize('start', {
+        regex: syntaxRules.keywordWithSpace(_ASK),
+        token: 'keyword',
+        next: 'gobble',
+      }),
+    ),
+  },
+  {
+    // Adds lists and 'at random'
+    name: 'level2',
+    rules: syntaxRules.pipe(syntaxRules.baseRules(_AT,_RANDOM),
+
+      syntaxRules.rule_printSpace(_PRINT,'expression_eol'),
+      syntaxRules.rule_isAsk(_IS,_ASK,'gobble'),
+      syntaxRules.rule_is(_IS,'gobble'),
+      syntaxRules.rule_turtle(_TURN,_FORWARD),
+      syntaxRules.rule_sleep(_SLEEP),
+
+    ),
+  },
+  {
+    // Adds quoted text
+    name: 'level3',
+    rules: syntaxRules.pipe(syntaxRules.baseRules(_AT,_RANDOM),
+      syntaxRules.rule_turtle(_TURN,_FORWARD),
+      syntaxRules.rule_printSpace('expression_eol'),
+      syntaxRules.rule_isAsk(_IS,_ASK),
+      syntaxRules.rule_is(_IS),
+    ),
+  },
+  {
+    // Adds if/else
+    name: 'level4',
+    rules: syntaxRules.pipe(syntaxRules.baseRules(_AT,_RANDOM),
+      syntaxRules.rule_printSpace(_PRINT),
+      syntaxRules.rule_isAsk(_IS,_ASK),
+      syntaxRules.rule_is(_IS),
+      syntaxRules.rule_ifElseOneLine(_IF,_ELSE,_IS,_IN),
+      syntaxRules.rule_expressions(_AT,_RANDOM),
+    ),
```

Comment:
This (and all other levels) are basically a copy-paste of en, could we somehow centralize this (if now, we can leave it as is)